File: solutions/javascript/raindrops/2/raindrops.js
//
// This is only a SKELETON file for the 'Raindrops' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

function rainDrop(val) {
let result = "";
val % 3 === 0 ? (result += "Pling") : "";
val % 5 === 0 ? (result += "Plang") : "";
val % 7 === 0 ? (result += "Plong") : "";
return result || val.toString();
}
export const convert = (val) => {
  return rainDrop(val)
};

File: solutions/go/raindrops/1/raindrops.go
package raindrops
import "strconv"

func Convert(n int) string {
	droplets:= ""
    if n % 3 == 0 {
    droplets+="Pling"
   }
   if n % 5 == 0 {
      droplets+= "Plang"
   }
   if n % 7 == 0 {
      droplets +="Plong"
    }
  if droplets == "" {
     return strconv.Itoa(n)
   }else{
    return droplets
  }
}
